content/en/docs/ambient/architecture/index.md

```diff
@@ -9,15 +9,66 @@ owner: istio/wg-networking-maintainers
+
+|  Name  | Feature Status | Scope | Description |
+| --- | --- | --- | --- | --- |
+| `istio.io/dataplane-mode` | Beta | `Namespace` |  Specifies the data plane mode, valid value: `ambient`. |
+| `istio.io/use-waypoint` | Beta | `Namespace` or `Service` or `Pod` | Enrolls your resource to use a given waypoint, valid values: `{waypoint-name}`, `{namespace}/{waypoint-name}`, or `#none` |
+| `istio.io/waypoint-for` | Alpha | `Gateway` | Specifies the waypoint's capture scope, valid value: `service` or `none` or `workload` or `all`. This label is optional and the default value is `service`. |
+
+In order for your `istio.io/use-waypoint` label value to be effective, you have to ensure the used waypoint is configured with the corresponding capture scope which is `service` by default. For example, when you label a pod to use a specific waypoint via the `istio.io/use-waypoint` label, the waypoint's capture scope should be `workload` or `all`.
+
+### Policy attachment to waypoints
+
+You can attach Layer 7 policies (such as `AuthorizationPolicy`, `RequestAuthentication`, `Telemetry`, `WasmPlugin`, etc) to your waypoint using `targetRef`.
+
+- To attach the entire waypoint, set `Gateway` as the `targetRef` value. The example below shows how to attach the `viewer` policy
+to the waypoint named `waypoint` for the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Gateway
+        group: gateway.networking.k8s.io
+        name: waypoint
+    {{< /text >}}
+
+- To attach a given service within the waypoint, set `Service` as the `targetRef` value. The example below shows how to attach
+the `productpage-viewer` policy to the `productpage` service in the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: productpage-viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Service
+        group: ""
+        name: productpage
+    {{< /text >}}
+
 ## Traffic routing
 
 In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
 1. **Uncaptured:** this is a standard pod without any mesh features enabled.
-1. **Captured:** this is a pod that has traffic intercepted by {{< gloss >}}ztunnel{{< /gloss >}}. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
+1. **Captured:** this is a pod that has traffic intercepted by {{< gloss >}}ztunnel{{< /gloss >}}. A pod's captured mode can be enabled by setting the `istio.io/dataplane-mode=ambient` label on its namespace, which enables all pods' captured mode for that namespace.
 1. **Waypoint enabled:** this is a pod that is "Captured" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed.
```

Comment:
```suggestion
1. **Waypoint enabled:** this is a pod that is "Captured" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed. With a waypoint, L7 policy can be enforced for pod traffic. Waypoints are enabled for a workload by setting the `istio.io/use-waypoint` label.
```

Reply:
I don't think it is correct to jump in to discuss pod/workload so fast given it is not enabled by default and is not the most common scenario.  Most common scenario is to enable waypoint for service.

Reply:
Yeah the last sentence is not meant to refer to workload-attached waypoints directly, but is just intended to be a generic lead into "here are the ways in which waypoints can be enabled for _your workloads_"

Reply:
Thanks for the clarification. I reworded it to not focus on pod:

```
To enforce L7 policies, add the `istio.io/use-waypoint` label to your resource to use waypoint for the labeled resource.
```